pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
     "qiskit[all]~=0.39.2",
     "pytket~=1.9.0",
     "mqt.bench@ git+https://github.com/cda-tum/MQTbench@dev",
+    "sb3-contrib>=1.7.0a0",
```

Comment:
I am all for using the latest stuff, but do we really need the `-contrib` package here (and an alpha version of that). Can't we just use the `stable-baselines3` package?

Reply:
The problem with that approach ist that `stable-baselines3` in its latest version (v1.6.2) uses `gym==0.21.0` because there are breaking changes in `v0.22.0`. The latest `gym` version is `v0.26.0`, which we also used in the training.

Reply:
Unfortunately, this does not work, because also the contrib alpha version depends on `gym==0.21.0`.
I am unsure whether it makes sense to adjust our Environment such that it is compatible again with that version. What do you think?

Reply:
Nah. Then just keep that in mind for future reference and once the package is out of alpha, we update.